app/views/repositories/_checkout_instructions.html.erb

```diff
@@ -1,30 +1,33 @@
-<% if instructions.present? && instructions.available? %>
-<form class="form -bordered -compressed">
-  <fieldset class="form--fieldset -collapsible">
-    <legend class="form--fieldset-legend" title="<%= l('repositories.checkout.instructions') %>"
-            onclick="toggleFieldset(this);"><%= l('repositories.checkout.instructions') %></legend>
-    <div class="form--field">
-        <label class="form--label"><%= l('repositories.checkout.url') %></label>
-        <div class="form--field-container">
-            <div class="form--field-affix">
-              <%= instructions.checkout_command %>
-            </div>
-            <div class="form--text-field-container -xwide">
-                <input type="text" class="form--text-field -clickable"
-                onclick="this.focus(); this.select()"
-                readonly="readonly" value="<%= instructions.checkout_url %>">
-            </div>
-        </div>
-        <div class="form--field-instructions -no-italic">
-          <%= simple_format instructions.instructions %>
-          <% if instructions.manages_permissions? %>
-            <p>
-              <span><%= l('repositories.checkout.access_permission') %></span>
-              <strong><%= l("repositories.checkout.access.#{instructions.permission}") %></strong>
-            </p>
-          <% end %>
+<div id="repository--checkout-instructions" class="notification-box -info">
```

Comment:
Yes, but with the recent announcement of chrome and firefox, this method will no longer work given the forced switch to 'click to play' mechanics of flash.

This does work in the next stable releases of Firefox, the current stables of IE and Chrome. Seems plenty to me and significantly better than having a flash dependency.